gprofiler/profilers/perf.py

```diff
@@ -245,32 +248,42 @@ def __init__(
         self._node_processes_attached: List[Process] = []
         self._perf_memory_restart = perf_memory_restart
         switch_timeout_s = duration * 3  # allow gprofiler to be delayed up to 3 intervals before timing out.
+        extra_args = []
+        if not is_profiler_disabled(perf_mode):
```

Comment:
Is this check even needed? If perf is disabled - we never get into the c'tor of this profiler, right?

Reply:
9f050e09ad2246e65b9ab3150ce310a3d3fb1e49